lexical-core/src/lib.rs

```diff
@@ -184,28 +184,6 @@ if #[cfg(all(feature = "correct", feature = "radix"))] {
 
 }   // lib
 
-// PANIC
-
-// Need to define a panic handler when we're not testing (panic handler
-// then becomes "unwind" but there is no_std). This causes us to fail
-// with doctests, so ensure `--tests` is passed to `cargo test` whenever
-// we are in a  `no_std` context.
-cfg_if! {
```

Comment:
This code is buggy but is needed for testing in a `no_std` environment. I've just checked and this works on nightly, so this should be removed. Can you submit this as a PR?

Reply:
Also, in your updated PR, please add the following line of code, removing `lang_items`, since it is no longer required (`core_intrinsics` is still required for math operations).

```diff
--- a/lexical-core/src/lib.rs
+++ b/lexical-core/src/lib.rs
@@ -116,7 +116,6 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 #![cfg_attr(all(not(feature = "std"), feature = "correct", feature = "radix"), feature(alloc))]
 #![cfg_attr(not(feature = "std"), feature(core_intrinsics))]
-#![cfg_attr(all(not(test), not(feature = "std")), feature(lang_items))]
```